src/chrome/content/rules/mosnet.ru.xml

```diff
@@ -1,6 +1,25 @@
-<ruleset name="mosnet.ru">
-	<target host="mosnet.ru" />
-	<target host="www.mosnet.ru" />
+<!--
+mosnet.ru ⁴
+www.mosnet.ru ⁴
+blocked.mosnet.ru ²
+em.mosnet.ru ³
+littlesister.mosnet.ru ⁴
+mail.mosnet.ru ⁴
+neo.mosnet.ru ⁴
+old.mosnet.ru ⁴
+orange.mosnet.ru ³
+privateoffice.mosnet.ru ⁴
+raven.mosnet.ru ²
+speedtest.mosnet.ru ²
+speedtestlocal.mosnet.ru ²
+
+
+² refused
+³ timed out
+⁴ self signed
```

Comment:
But we [don't accept domains with chain issues](https://github.com/EFForg/https-everywhere/pull/3279#issuecomment-160349032), isn't it?